client/app/scripts/superdesk-highlights/views/settings.html

```diff
@@ -62,25 +62,26 @@ <h3 ng-show="configEdit._id" translate>Edit highlight configuration</h3>
 								ng-selected="configEdit.auto_insert === getHourVal(hour)" translate>Last {{ hour }} hours</option>
 						</select>
 					</div>
-                    <div class="field col">
-                        <a class="btn btn-info add"
+                    <div class="field">
+                    	<label translate>Package groups</label>
+                        <button class="btn btn-info add"
                             sd-focus-element data-target="#insert-group"
                             ng-click="editGroup(''); selectedGroup = null"
                             ng-disabled="editingGroup">
                             <i class="icon-plus-small icon-white"></i>
-                        </a>
-                        <div class="header" translate>Group Packages</div>
+                            Add group
+                        </button>
                         <div class="content">
-                            <div class="list clearfix">
-                                <div ng-repeat="group in configEdit.groups">
-                                    <span ng-if="isActiveGroup(group)" sd-character-count data-item="editingGroup.name" data-limit="limits"></span>
-                                    <div ng-click="selectGroup(group)"
-                                        ng-class="{active: selectedGroup === group, editable: isActiveGroup(group)}">
-                                        <span ng-if="!isActiveGroup(group)">{{ group }}
-                                        </span>
+                            <ul class="pills-list pills-editable clearfix">
 
-                                        <span ng-if="isActiveGroup(group)">
-                                            <input type="text" id="edit-group" ng-model="editingGroup.name" ng-keyup="handleGroupEdit($event);" />
+                                <li ng-repeat="group in configEdit.groups"
+                                   	ng-class="{editable: isActiveGroup(group)}">
+
+                                        <h6 ng-if="!isActiveGroup(group)">{{ group }}</h6>
+
+										<span ng-if="isActiveGroup(group)" sd-character-count data-item="editingGroup.name" data-limit="limits"></span>
+                                        <span>
+                                            <input type="text" ng-if="isActiveGroup(group)" id="edit-group" ng-model="editingGroup.name" ng-keyup="handleGroupEdit($event);" />
```

Comment:
Do we really want/need to invoke the event handler on every `keyup` event? Also, was the _keyup_ event chosen for a reason instead of e.g. _onchange_?